usr.sbin/pkg/pkg.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+#
+# Copyright (c) 2022 Konrad Witaszczyk
+#
+# This software was developed by SRI International and the University of
+# Cambridge Computer Laboratory under DARPA/AFRL contract (FA8750-10-C-0237)
+# ("CTSRD"), as part of the DARPA CRASH research programme.
+#
+# This software was developed by SRI International and the University of
+# Cambridge Computer Laboratory (Department of Computer Science and
+# Technology) under DARPA contract HR0011-18-C-0016 ("ECATS"), as part of the
+# DARPA SSITH research programme.
```

Comment:
Don't need either of these sponsor ACKs (but maybe do need a DSbD UKRI one? depends where your salary comes from)

Reply:
Most likely the right license header ACK for @kwitaszczyk is something like:

```
 * This software was developed at the University of Cambridge Department of
 * Computer Science and Technology with support from Innovate UK project
 * 105694, "Digital Security by Design (DSbD) Technology Platform Prototype".
```

Reply:
Thanks!